src/v2/models/AppState.js

```diff
@@ -23,15 +23,12 @@ export default Model.extend({
     factor: 'object',      // optional
     user: 'object',        // optional
     currentFormName: 'string',
+    idx: 'object',
+    remediations: 'array',
+    __previousResponse: 'object'
   },
 
   derived: {
-    remediation: {
-      deps: ['currentState'],
```

Comment:
Is `currentState` still being used anywhere else? If not, we should remove the definition.